tools/git.md

```diff
@@ -35,6 +35,9 @@ http://stackoverflow.com/questions/14290113/git-pushing-code-to-two-remotes#answ
 
 `git remote set-url --add`
 
+### git blame a diff
```

Comment:
This should be in the previous commit https://github.com/OpenDataServices/developer-docs/pull/50/commits/2854d48c2dea75b3c7f0eac02ea86da3dd79a8e6